### DISCUSSION TITLE
[Problem]Create multiple roles via github login

### DISCUSSION THREAD

**Discussion by matheusnagatsuka** [gravitational/teleport#14647] [Q&A] [answered]
I have 5 development environments and would like to merge them into one teleport and still be able to log in through github. Can't I create access rules for certain users that log in through git? 
I tried to manage through github teams but it was not possible.
Can someone assist me?

**Comment by webvictim** [comment, marked as answer]
If you run the Teleport `ssh_service` on each machine that you're trying to provide access to, you can set labels for each which will show up in the web UI.

You can then create a role which grants access to certain labels (via `node_labels`) and certain logins (via `logins`)

You should then be able to use the `teams_to_roles` mapping in the Github connector to assign a given Github team to one or more roles, thus granting access.
